tests/execution/grpc-oneof.md

```diff
@@ -169,9 +173,9 @@ type oneof__Response__Var2 @tag(id: "oneof.Response") {
     query: >
       query {
         oneof__OneOfService__GetOneOfVar1(request: { command: { command: "start" } }) {
-          # TODO: check that it's possible to get shared field from Union like that
-          # outside of the fragment
-          usual
```

Comment:
this is wrong way to select the shared field. It doesn't work for graphql-js and static async-graphql and seems like possible due to bug in the dynamic implementation